src/components/static/upcoming/Upcoming.jsx

```diff
@@ -18,7 +18,7 @@ const Upcoming = () => {
       <div className="flex h-[72%] flex-row overflow-hidden justify-between bg-gradient-to-b from-design-green-100 via-design-green-100 to-design-green-400">
         <Image className="w-5/6 md:w-3/12 -ml-56 mt-52" src={Mushrooms2} />
         <div className="absolute w-5/6 md:w-[55%] flex flex-col left-[8%] md:left-[7%] object-contain">
-          <div className="flex rounded-3xl md:h-[60%] mt-8 items-center md:items-start bg-design-green-500/70 p-10 flex-col">
+          <div className="flex rounded-3xl md:h-[60%] mt-12 items-center md:items-start bg-design-green-500/70 p-10 flex-col">
```

Comment:
oh wait, this is the change from your other branch on another issue. Can you remove this? I can merge your code for this issue then. The other issues needs some fixes.